src/formats/af.h

```diff
@@ -20,17 +20,17 @@
  * Contains information about a single HAR (combat robot).
  */
 typedef struct {
-    uint16_t file_id;     ///< File ID
-    uint16_t exec_window; ///< Move execution window (?)
-    float endurance;      ///< HAR Endurance
-    uint8_t unknown_b;    ///< Unknown value
-    uint16_t health;      ///< HAR Health
-    float forward_speed;  ///< HAR fwd speed
-    float reverse_speed;  ///< HAR bwd speed
-    float jump_speed;     ///< HAR jump speed
-    float fall_speed;     ///< HAR fall speed
-    uint8_t unknown_c;    ///< Unknown value
-    uint8_t unknown_d;    ///< Unknown value
+    uint16_t file_id;      ///< File ID
+    uint16_t exec_window;  ///< Move execution window (?)
+    uint32_t endurance;    ///< HAR Endurance
```

Comment:
I think the original game has endurance/stun go negative, so uint may not be a good idea here.

Reply:
uint32_t was chosen here to match the `sd_read_udword` that af.c uses to read this value.

I'll make har_endurance_t an int32_t, and change this to be an int32_t and use `sd_read_dword` (bonus: making this signed lets me nuke the saturating sub helper).